### PR TITLE
MODSOURMAN-924 Mapping bib's $9 into subjects, series, alternativeTitles fields

### PR DESCRIPTION
**Pull request by psmagin**
## Purpose
Mapping bib's $9 into subjects, series, alternativeTitles fields

### CODE REVIEW

**Review by sherzodn — APPROVED** [review]

**Review by sherzodn — COMMENTED** [review]
tests are failed

File: mod-source-record-manager-server/src/main/resources/rules/marc_bib_rules.json [hunk 1]
```diff
@@ -5653,328 +5653,352 @@
...
+            {
+              "value": "--",
+              "subfields": []
+            }
           ]
         },
         {
-          "value": "--",
-          "subfields": []
+          "target": "series.authorityId",
```
Comment (RomanChernetskyi) [inline mod-source-record-manager-server/src/main/resources/rules/marc_bib_rules.json:5735]: Hello, @psmagin, should it be subjects. authorityId?

Comment (psmagin) [reply mod-source-record-manager-server/src/main/resources/rules/marc_bib_rules.json]: Thanks. Fixed

**Review by viacheslavkol — APPROVED** [review]

**Review by Shans-Kaluhin — APPROVED** [review]
Waits for [data-import-processing-core v4.0.0-SNAPSHOT](https://github.com/folio-org/data-import-processing-core/pull/248)

**Pull request merged**